Lib/glyphsLib/builder/axes.py

```diff
@@ -141,7 +141,13 @@ def to_designspace_axes(self):
     assert isinstance(regular_master, classes.GSFontMaster)
 
     custom_mapping = self.font.customParameters["Axis Mappings"]
-
+    if not custom_mapping:
+        logger.warning(
```

Comment:
hm warning that may be too noisy. I'd suggest an INFO level, and much shorter than this, ideally one line.

Reply:
How about:

''Axis Mappings' customParameter not found. Add this parameter to define an explicit mapping."


Personally, I prefer longer logs in these tools because most users won't dig into the source code so it may be beneficial for them to know what's happening. Perhaps some simple docs would be better.

Reply:
i think documentation doesn't belong in the logging. There is nothing technically wrong with one using Axis Locations or the instance inference as they are working now, if one doesn't need the extra customization provided by Axis Mappings, so the WARNING is too much. Even INFO seems out of place for me.
This new feature can be sort of experimental at the beginning, at least until things settle and Glyphs.app has implemented it in a similar way.
I suggest we remove any such recommendations, but work on documenting how this Axis Mappings is supposed to work and put a link somewhere. Adding full automated sphinx/ReadTheDocs etc. support for glyphsLib would be quite an effort and haven't got cycles for that now.

Reply:
How about we just drop the logging of this feature completely for the time being? or we could simply do logger.info("Using 'Axis Mapping' customParameter")?

Reply:
yes, that's what I'm say. Let's not log anything for now. Or if you like just log what method is being used to obtain the user:design mapping among the three supported ones.

Reply:
Thanks Cosimo. I'll rip it out.

Re docs, sorry I didn't mean a full blown sphinx setup. I'm more interested in a high level overview that explains how the full fontmake stack works. Such a doc would help most font devs who are not programmers diagnose common issues or at least understand where problems may occur.